plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -138,13 +138,29 @@ public function track_product_updated_client_side( $post ) {
 						return;
 					}
 
+					var tagsText          = $( '[name=\"tax_input[product_tag]\"]' ).val();
 					var currentStockValue = $( '#_stock' ).val();
+					var description_value  = $( '#content' ).is( ':visible' ) ? $( '#content' ).val() : tinymce.activeEditor.getContent();
```

Comment:
One more minor thing- this only works if the editor has been clicked on.  I think we'll need to use `.get()` to make this work in case they don't update the content.

```suggestion
					var description_value  = $( '#content' ).is( ':visible' ) ? $( '#content' ).val() : tinymce.get( 'content' ).getContent();
```

Reply:
Good idea! Sorry for that. I added it in the commit eef164b.
@joshuatf could you take another look at this pr?